src/extension/sldService/src/main/java/org/geoserver/sldservice/utils/classifier/RasterSymbolizerBuilder.java

```diff
@@ -418,4 +472,12 @@ private NumberRange getOperationRange(ImageWorker iw) {
             }
         }
     }
+
+    private String getPercentagesLabelPortion(double[] percentages, int i) {
+        StringBuilder sb = new StringBuilder();
```

Comment:
Overly complicated:

```
if (percentages == null) return "";
return " (" + percentages[i] + ")";
```

Reply:
By the way, the vector code states it's a percentage, e..g adds " (x%)" while this one just adds " (x)". I think it's best to have the % sign, the number per se means nothing to the ones just looking at the map without knowing how the the SLD classification service worked, but at the very least, they should be consistent.

Reply:
fixed